.github/workflows/main.yaml

```diff
@@ -3,7 +3,7 @@ name: CI
 on: ["push"]
 
 jobs:
-  build-and-push:
+  test:
     runs-on: ubuntu-latest
     timeout-minutes: 80
     environment: ${{ github.ref == 'refs/heads/master' && 'CD' || 'CI' }}
```

Comment:
`      - name: Set variables` section should be deleted for a cleaner definition, normally it looks like it sets stuff used in published artifacts, but in this particular repo all those BUILD_DATE,IMAGE_REVISION,DOCKER_IMAGE_ seem to be unused